src/master.rs

```diff
@@ -517,9 +517,97 @@ impl Master {
         Ok(time)
     }
 
+    pub fn foe_read(&mut self, idx: u16, name: String) -> Result<Vec<u8>> {
```

Comment:
`idx` should be `SlavePos`
`name` should be `&str`

Reply:
Yes, good idea. Fixed it